code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1757,6 +1757,43 @@
 				continue
 			movable_content.wash(clean_types)
 
+/datum/reagent/medicine/radaway
+	name = "RadAway"
+	description = "A potent but toxic chemical solution that binds with radioactive particles and render them inert. Applying this through spray or smoke will cleanse contaminanted surfaces."
+	color = "#9f5a2f"
+	var/old_insulation = RAD_NO_INSULATION
+	taste_description = "metallic dust"
+	self_consuming = TRUE
+
+/datum/reagent/medicine/radaway/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	if(method == TOUCH || method == VAPOR)
+		M.wash(CLEAN_RAD) //you only get decontaminated if it's spray based, can't spam out 100 1u pills
+	
+/datum/reagent/medicine/radaway/on_mob_life(mob/living/carbon/M)
+	if(M.radiation > 0)
+		M.radiation -= min(M.radiation, 15)
```

Comment:
this isn't contamination and we have other radiation removing chemicals like seiver

Reply:
took out the rad removal part